CurrentTimeApp.Components/CurrentTimeApp.Components.csproj

```diff
@@ -15,7 +15,7 @@
 	</ItemGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="7.0.0" />
+		<PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="7.0.1" />
```

Comment:
This dependency will be updated with https://github.com/jjliggett/CurrentTimeApp/pull/85